modules/ipi-install-provisioning-the-bare-metal-node.adoc

```diff
@@ -60,44 +64,50 @@ openshift-worker-1.example.com      1         1         1       1           55m
 
 . Increase the number of worker nodes by one.
 +
-[source,bash]
+[source,terminal,subs="+quotes"]
 ----
 $ oc scale --replicas=<num> machineset <machineset> -n openshift-machine-api
 ----
 +
-Replace `<num>` with the new number of worker nodes. Replace `<machineset>` with the name of the machine set from the previous step.
+where:
+`<num>`:: Specifies the new number of worker nodes. 
```

Comment:
Should this not read 'Specifies the number of new worker nodes.' 
I suppose it depends on whether the nodes themselves are new or if there is a pool of existing nodes and the number of nodes is adjusted (new), as needed.

Reply:
Updated as suggested.